Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5443,6 +5443,7 @@ OptInPartitionedCookiesEnabled:
   category: dom
   humanReadableName: "Opt-in partitioned cookies (CHIPS)"
   humanReadableDescription: "Enable opt-in partitioned cookies"
+  condition: HAVE(ALLOW_ONLY_PARTITIONED_COOKIES)
```

Comment:
With this change, I think we need to move related functions (like`isOptInCookiePartitioningEnabled()`) under condition `if HAVE(ALLOW_ONLY_PARTITIONED_COOKIES)`

Reply:
We don't need to do that, it will default to false, but I made this change so it is clear.